R/CreatePackageReport.R

```diff
@@ -17,7 +20,8 @@
 CreatePackageReport <- function(packageName
                                 , packageReporters = DefaultReporters()
                                 , packagePath = NULL
-                                , reportPath = file.path(getwd(),paste0(packageName,"_report.html"))) {
+                                , reportPath = file.path(getwd(),paste0(packageName,"_report.html"))
+                                , orphanNodeClusteringThreshold = Inf) {
```

Comment:
This is not the right way to do this. We shouldn't be moving data through the package via this function signature. IMHO, `orphanNodeClusteringThreshold` should be in the constructor of the relevant graph reporters and stored as a private member of those reporters. Whenever you find yourself saying "if the reporter is of this one special class" inside `CreatePackageReport`, it's a good sign that something is being added to this function that shouldn't be

Reply:
@jameslamb `orphanNodeClusteringThreshold` is in fact a member of `AbstractGraphReporter` with a default value of `inf`.  

`CreatePackageReport` is supposed to be a one-stop-shop function for creating a package report.  If we don't surface parameters here in this function, you're saying it shouldn't be configurable by the user?